docs/explanation/stable-release-updates.rst

```diff
@@ -1,2 +1,257 @@
+    verify the fix.
+  - **Where Problems Could Occur**: Highlight potential areas where
+    :ref:`regressions <Regressions>` might happen. This section should show that
+    potential risks have been considered. It should also provide additional test cases
+    to ensure there are no :ref:`regressions <Regressions>`.
+
+5. Prepare the :term:`SRU` upload, attach a ``debdiff`` to the bug, and request sponsorship
+   by subscribing ``ubuntu-sponsors`` to the bug. The upload should have the correct
+   release in the changelog header, a detailed and user-readable changelog and no
+   unrelated changes. If you can upload directly, use ``dput`` as normal. Once uploaded,
+   change the bug status to ``In Progress``. The status will be automatically updated to
+   ``Fix Committed`` once accepted into ``release-proposed``.
+6. Ensure that the version number doesn't conflict with any future versions in other
+   :ref:`Ubuntu releases <UbuntuReleases>`. Also, include a reference to the :term:`SRU` bug
+   number in the changelog using the ``LP: #NNNNNN`` format, and only reference public bugs.
+7. Once the :term:`Ubuntu SRU Team` reviews and accepts your upload, test the binaries
+   in the :term:`Ubuntu Archive` and follow up in the bug report with your verification
+   results. The :term:`Ubuntu SRU Team` will evaluate the testing feedback and move the
+   package into :ref:`updates <ArchivePockets_Updates>` after it passes a minimum aging
+   period of 7 days.
+8. Subscribe to the bugmail of the package in :term:`Launchpad`, and monitor
+   :term:`Launchpad` for bug reports relating to the update for at least one week. If
+   you notice and confirm any regression, document it in a bug report marked with an
+   ``Importance: critical`` label.
+
+SRU phasing
+-----------
+
+Once a package is released to :ref:`updates <ArchivePockets_Updates>`,
+the update is then phased so that the update is gradually made available to expanding
+subsets of Ubuntu users.
+
+Initially, the ``Phased-Update-Percentage`` is set to 10%, with a job running every 6
+hours to monitor for :ref:`regressions <Regressions>`. If no issues are detected, the
+update percentage increments by 10% until it reaches 100%. So an update will become
+fully phased after 54 hours. If a regression is found, the update is halted and the
+``Phased-Update-Percentage`` is set to 0%. This will then cause supported package
+managers not to install the update.
+
+Investigating halted phased updates
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To investigate why phasing stopped, use the phased updates report.
+
+When investigating an increased rate of crashes, focus on the crashes with the highest
+number of occurrences. Examine the occurrences table to determine if these crashes are
+happening more frequently with the updated version of the package. If they are,
+investigate the cause and address the crash in a follow-up :term:`SRU`. If not,
```

Comment:
nitpick: This page already established the acronym. There is no value in linking to the glossary.

```suggestion
investigate the cause and address the crash in a follow-up SRU. If not,
```